code/controllers/subsystems/vote.dm

```diff
@@ -247,20 +255,20 @@ var/datum/controller/subsystem/vote/SSvote
 				if(ROUNDTYPE_STR_MIXED_SECRET in choices)
 					AddChoice(ROUNDTYPE_STR_MIXED_SECRET, "Mixed Secret")
 			if("crew_transfer")
-				if(check_rights(R_ADMIN|R_MOD, 0))
-					question = "End the shift?"
-					AddChoice("Initiate Crew Transfer")
-					AddChoice("Continue The Round")
-				else
-					if (get_security_level() == "red" || get_security_level() == "delta")
+				if(!check_rights(R_ADMIN|R_MOD, 0))
```

Comment:
Let's try this again, this time with feeling.
This is undocumented in the actual body of the PR, and is a weird change that will result in much longer rounds, as well as antags being able to pretty much instantly deny a transfer at the 1:59 mark. Does that satisfy you, Arrow?

Reply:
Antags can already do that... people who use the CT mechanics as a ghetto evac vote should be ashamed.